glanzb/homework1.md

```diff
@@ -0,0 +1,183 @@
+### Homework #1
+(Due Monday 1/12)
+
```

Comment:
You did really well on this section of the homework, but I think you forgot to test for var x = NaN; on a few of the questions.